hypervisor/arch/x86/guest/ucode.c

```diff
@@ -48,7 +47,6 @@ void acrn_update_ucode(struct acrn_vcpu *vcpu, uint64_t v)
 	int32_t err;
 	uint32_t err_code;
 
-	spinlock_obtain(&micro_code_lock);
```

Comment:
Add a comment to say the user (guest operating system) should grantee it won't issue 2nd  microcode update comment when the 1st microcode update is on going..